lib/search.rb

```diff
@@ -1151,7 +1144,7 @@ def self.ts_query(term: , ts_config:  nil, joiner: nil, weight_filter: nil)
 
   def self.to_tsquery(ts_config: nil, term:, joiner: nil)
     ts_config = ActiveRecord::Base.connection.quote(ts_config) if ts_config
-    tsquery = "TO_TSQUERY(#{ts_config || default_ts_config}, '#{self.escape_string(term)}')"
+    tsquery = "TO_TSQUERY(#{ts_config || default_ts_config}, #{Search.wrap_unaccent("'#{self.escape_string(term)}'")})"
```

Comment:
Just to get rid of string interpolation within string interpolation: 😃 
```suggestion
    escaped_term = Search.wrap_unaccent("'#{self.escape_string(term)}'")
    tsquery = "TO_TSQUERY(#{ts_config || default_ts_config}, #{escaped_term})"
```